pkg/client/fake/client_test.go

```diff
@@ -2554,6 +2538,51 @@ var _ = Describe("Fake client", func() {
 		wg.Wait()
 	})
 
+	It("supports server-side apply of a client-go resource", func() {
```

Comment:
Should we have test coverage that tests that managedFields are returned? (and only if the flag is set)

Reply:
Added

Reply:
Thx!